universal-login-relayer/lib/services/WalletService.ts

```diff
@@ -19,8 +19,8 @@ class WalletService {
   async create(key: string, ensName: string, overrideOptions = {}) {
     const ensArgs = this.ensService.argsFor(ensName);
     if (ensArgs !== null) {
-      let args = [key, ...ensArgs];
-      const initData = new utils.Interface(WalletMasterContract.interface).functions.initializeWithENS.encode(args);
+      let args = [key, ...ensArgs] as string[];
```

Comment:
In `ensService.ts`:
`argsFor(ensName: string) {` -> `argsFor(ensName: string) : string[] {`